alws/utils/pulp_client.py

```diff
@@ -572,8 +572,25 @@ async def get_rpm_packages(
                 exclude_fields=exclude_fields,
                 **search_params,
             )
-            all_rpms.extend(result["results"])
-        return all_rpms
+            all_entities.extend(result["results"])
+        return all_entities
+
+    async def get_rpm_packages(
+        self,
+        include_fields: typing.List[str] = None,
+        exclude_fields: typing.List[str] = None,
```

Comment:
actually, it's should be `Optional[List[str]] = None`, because if we expect list, it cannot be None